python/lsst/pipe/base/execution_reports.py

```diff
@@ -284,17 +309,26 @@ def to_summary_dict(self, butler: Butler, do_store_logs: bool = True) -> dict[st
             The Butler used for this report.
         do_store_logs : `bool`
             Store the logs in the summary dictionary.
+        human_readable : `bool`
+            Store more human-readable information to be printed out to the
+            command-line.
 
         Returns
         -------
         summary_dict : `dict`
             A dictionary containing a summary of a `TaskExecutionReport` for
             each task in the quantum graph.
         """
-        return {
-            task: report.to_summary_dict(butler, do_store_logs=do_store_logs)
```

Comment:
Why not just change this line to be
`task: report.to_summary_dict(butler, do_store_logs=do_store_logs, human_readable=human_readable)` 

instead of adding the separate if/else statement?

Reply:
Thanks! I was thinking too hard about that one.